core/src/main/java/org/hisp/dhis/android/core/settings/ProgramConfigurationSetting.java

```diff
@@ -92,6 +98,10 @@ public abstract static class Builder {
 
         public abstract Builder itemHeader(ProgramItemHeader itemHeader);
 
+        public abstract Builder minimumLocationAccuracy(Integer minimumLocationAccuracy);
+
+        public abstract Builder disableManualLocation(Boolean disableManualLocation);
```

Comment:
Does this property a default value or is it nullable? Should it has a default value?

Reply:
I think it should be enabled by default.

Reply:
Most of the properties here are nullable and are not given a default value in the SDK. They have a default initial value in the ASWA (the initial config) and the application gives them a default value when they are null. I would keep the same behavior for this property. 
We could consider that the SDK should be the responsible for setting the default value, but if we do so, it should be done for all the properties. So I would keep this property nullable for now